CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 ### Features
 
+- We now generate the ts bindings from the same uniffi code that swift and kotlin use.
+
+  - Arrays are now passed as ArrayBuffer between client and the FFI layer, changing parameter and return types.
+  - `CustomConfiguration.keyRotationSpan` now defines milliseconds instead of seconds
```

Comment:
Is there really no other type where we define a `Duration` in the FFI?

Reply:
In the FFI this is the only occurrence.